cmd/rm-main.go

```diff
@@ -156,8 +182,15 @@ func checkRmSyntax(ctx context.Context, cliCtx *cli.Context, encKeyDB map[string
 	isRecursive := cliCtx.Bool("recursive")
 	isStdin := cliCtx.Bool("stdin")
 	isDangerous := cliCtx.Bool("dangerous")
+	isVersions := cliCtx.Bool("versions")
+	versionID := cliCtx.String("version-id")
 	isNamespaceRemoval := false
 
+	if isVersions && versionID != "" {
```

Comment:
to keep things simple, we should fail even if `rewind` is specified when `--version-id` flag is specified. If `version-id` flag is specified, neither `rewind` nor `versions` flags are allowed.

Reply:
nor `recursive` flag

Reply:
> to keep things simple, we should fail even if `rewind` is specified when `--version-id` flag is specified. If `version-id` flag is specified, neither `rewind` nor `versions` flags are allowed.

Done